torch/csrc/jit/tensorexpr/tensorexpr_init.cpp

```diff
@@ -830,7 +830,7 @@ void initTensorExprBindings(PyObject* module) {
             value_ptrs.reserve(py::len(values));
             for (const auto& value : values) {
               if (py::isinstance<py::int_>(value)) {
-                value_ptrs.emplace_back(value.cast<int64_t>());
+                value_ptrs.emplace_back(value.cast<int>());
```

Comment:
Again, you cannot assume `py::int_` is `int`. https://github.com/pybind/pybind11/blob/80dc998efced8ceb2be59756668a7e90e8bef917/include/pybind11/pytypes.h#L1722-L1734

Reply:
Thanks, didn't know about that. Will update it.

Reply:
Would it be fine here to loop not only though `values`, but through `self.buffer_args()` at same time? In that case on big endian systems for `py::int_` value can be casted to correct type, correctly stored in buffer, and later correctly read.

The problem with calculating correct offset in `SimpleIREvaluator::bindArg` is that now `void *data` is pointer to some buffer, but no size of buffer is passed. Either have to correctly fill this buffer from the start or add size of buffer as argument as well. Or just assume it's always 8 bytes, but that doesn't sound good to me.